lib/mix/tasks/ecto.dump.ex

```diff
@@ -46,11 +47,20 @@ defmodule Mix.Tasks.Ecto.Dump do
     * `-q`, `--quiet` - run the command quietly
     * `--no-compile` - does not compile applications before dumping
     * `--no-deps-check` - does not check dependencies before dumping
+    * `--migration-prefixes` - comma-separated list of prefixes that will have their migration table exported. Defaults to `public`. (Postgres only)
   """
 
   @impl true
   def run(args) do
     {opts, _} = OptionParser.parse! args, strict: @switches, aliases: @aliases
+
+    opts =  if Keyword.has_key?(opts, :migration_prefixes) do
+      {_, opts} = Keyword.get_and_update(opts, :migration_prefixes, &{&1, String.split(&1, ",")})
+      opts
+    else
+      opts
+    end
```

Comment:
```suggestion
    opts = Keyword.update(opts, :migration_prefixes, nil, &String.split(&1, ","))
```

I'm pretty sure this is enough because missing/`nil` will be treated the same:

```elixir
migration_prefixes = config[:migration_prefixes] || ["public"]
```